packages/core/codegen.ts

```diff
@@ -17,7 +17,9 @@ const getOperationName = (document: DocumentNode) => {
 
 const config: CodegenConfig = {
   overwrite: true,
-  errorsOnly: true,
+  errorsOnly: false,
+  debug: true,
+  verbose: true,
```

Comment:
enables more logs related to the schema generation by default. 
we receive some issues related to the generic message showed previously.